src/utils/db.js

```
@@ -0,0 +1,11 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST || 'localhost',
+  username: process.env.DB_USERNAME || 'postgres',
+  password: process.env.DB_PASSWORD || '123456',
+  database: process.env.DB_DATABASE || 'postgres',
+  port: Number(process.env.DB_PORT) || 5432,
```

Right now default credentials (e.g., password '123456') are used when env vars are missing. Consider validating required environment variables at startup and failing fast in production, or log a clear warning when using defaults so misconfiguration is easier to detect.